libs/design-system/styles/_icons.scss

```diff
@@ -1,5 +1,5 @@
-@use '@fontsource-variable/material-symbols-outlined' as outlined;
-@use '@fontsource-variable/material-symbols-rounded' as rounded;
+@use '@fontsource-variable/material-symbols-outlined/full.css' as outlined;
```

Comment:
What is missing from the default import? Including full.css greatly increases the assets bundle size